lib/Db/PersonMapper.php

```diff
@@ -38,6 +38,22 @@ public function __construct(IDBConnection $db) {
 		parent::__construct($db, 'face_recognition_persons', '\OCA\FaceRecognition\Db\Person');
 	}
 
+
+	public function find (string $userId, int $personId): Person {
```

Comment:
this can return Person|null, right? I think PHP cannot have nullable return type (I was struggling with that, if I remember corectly). If this can return null, maybe you should ignore return type and just mention that in docstring that @return is "Person|null" (I am telling you this, as I think you will get runtime error from php type system if you return null here? (I might be wrong)

Reply:
findEntity not return null.. In any case return an exception 'DoesNotExistException' .. :disappointed: 

If you want I manage this same exception here, to return null, but considering that it is a find method that needs an **known personId** -Almost always obtained by some other query- should not fail. If we pass a nonexistent id, I understand that we would be using this query badly.

> @ return is "Person|null" 
Is introduces on PhP 7.1 as ': ?Person'

http://php.net/manual/en/migration71.new-features.php